common/config/environment.go

```diff
@@ -27,11 +27,13 @@ const (
 	DDInfraOSFamily          = "osFamily"
 
 	// Agent Namespace
-	DDAgentDeployParamName        = "deploy"
-	DDAgentVersionParamName       = "version"
-	DDAgentFullImagePathParamName = "fullImagePath"
-	DDAgentAPIKeyParamName        = "apiKey"
-	DDAgentAPPKeyParamName        = "appKey"
+	DDAgentDeployParamName               = "deploy"
+	DDAgentVersionParamName              = "agentVersion"
```

Comment:
We probably don't need to rename, we have `ddagent:version`, having `ddagent:agentVersion` is a bit redundant.